frontend/public/kubevirt/components/modals/start-stop-vm-modal.jsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import * as _ from 'lodash';
 
 import { PromiseComponent } from '../utils/okdutils';
 import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory/okdfactory';
-import { k8sPatch } from '../../../module/k8s';
+import { k8sPatch, k8sUpdate } from '../../../module/k8s';
+import { ANNOTATION_FIRST_BOOT } from 'kubevirt-web-ui-components/dist/js';
```

Comment:
We should avoid the `dist/js` in the import.
Can you please verify, if https://github.com/kubevirt/web-ui-components/pull/68 fixes that?

Same occurrence is in `vm.jsx`.

Reply:
that helps!